docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       - ./backend/db-setup/.pgpass/:/usr/local/lib/.pgpass
       - ./backend/db-setup/snapshots/:/usr/local/lib/snapshots/
       - ./backend/db-setup/generate_db_data.py:/usr/local/lib/generate_db_data.py
-    command: -p ${SR_DB_PORT:-5432}
+    command: -p ${SR_DB_PORT:-5432} -c log_statement=all
```

Comment:
This logs all the query's issued to the db, you can access these logs inside the db container

use the command `docker logs db -f -n 10` to view the queries in real-time.
This only affect local dev, I dont see any harm keep it on, let me know what you think.

Reply:
I don't think it's an exaggeration to say I've wanted this functionality for years 🤣 As long as this only affects local dev, I think it's great! 

Could you add some wiki documentation for the `docker logs` piece?

Reply:
Done! https://github.com/CDCgov/prime-simplereport/wiki/Log-local-DB-queries